packages/core/package.json

```diff
@@ -13,17 +13,19 @@
   "scripts": {
     "generate:schema": "tsx src/server/generator/generateGraphQLSchemaFile.ts",
     "generate:codegen": "graphql-codegen",
-    "generate:copy-back": "copyfiles \"@generated/**/*\" $DESTINATION",
-    "generate": "faststore generate-graphql -c -d",
```

Comment:
I miss the -d (debug) flag. 
Historically, some clients complain about generic errors in the generated graphql process. This debug flag adds more visibility into the graphql generation flow. What do you think?

Reply:
I belive that the flag and better error messages are two different problems, but I'll add this back

Reply:
I linked these two topics because we previously saw the specific error messages when using the flag.